src/utils/usageReportConsent.js

```diff
@@ -7,3 +7,7 @@ export const getConsentStatus = () => {
 export const setConsentStatus = consent => {
   localStorage.setItem(KEY, !!consent)
 }
+
+export const getConsentConfirmationStatus = () => {
```

Comment:
Please rethink the name of this function. This is a getter for whether instrumentation consent has been answered.

Reply:
How about one of these: `hasUserConsented` or `isConsentSet` or `getConsentAnswer`?

There are only [two hard things](https://martinfowler.com/bliki/TwoHardThings.html)...